### PR TITLE
Release/v1.1

### PR DESCRIPTION
**Pull request by jibola1997**

### CODE REVIEW

**Review by BadSingleton — APPROVED** [review]

**Review by BadSingleton — APPROVED** [review]

**Review by BadSingleton — APPROVED** [review]

File: plugins/basic/bootstrap.py [hunk 1]
```diff
@@ -66,11 +66,10 @@ def plugin_startup():
     except:
         import traceback
         
-        # Only log in the client as we might have gotten a "connection reset" 
-        # exception
-        import sg_client
-        sg_client.log('Shotgun Toolkit Error:')
-        sg_client.log('Exception stack trace:\n\n{}'.format(traceback.format_exc()))
+        # Only log in bootstrap incase of an exception
```
Comment (lassond) [inline plugins/basic/bootstrap.py:69]: "Only log" does not apply here. This was a workaround for connection issues with rpyc.
Please remove this comment as it is confusing

File: plugins/basic/sg_bootstrap.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 """
-The Unity client module for Shotgun. Responsible of customizing the client 
+The Unity bootstrap module for Shotgun. Responsible of customizing the bootstrap
```
Comment (lassond) [inline plugins/basic/sg_bootstrap.py:2]: Please remove the second sentence. It does not make sense now that we moved out of rpyc

File: plugins/basic/sg_bootstrap.py [hunk 3]
```diff
@@ -21,10 +21,10 @@ def log(msg):
 
     print(message)
 
-# We only initialize Shotgun once in the client process
+# We only initialize Shotgun once in the bootstrap process
```
Comment (lassond) [inline plugins/basic/sg_bootstrap.py:24]: There is no such thing as a bootstrap process. Just keep `We only initialize Shotgun once`

File: plugins/basic/sg_bootstrap.py [hunk 4]
```diff
@@ -63,7 +63,7 @@ def bootstrap_shotgun():
     """
     global _shotgun_is_initialized
     if _shotgun_is_initialized:
-        log('Shotgun is already initialized in the client process')
+        log('Shotgun is already initialized in the bootstrap process')
```
Comment (lassond) [inline plugins/basic/sg_bootstrap.py:66]: Change to `Shotgun is already initialized`

File: plugins/basic/sg_bootstrap.py [hunk 5]
```diff
@@ -88,9 +88,9 @@ def bootstrap_shotgun():
         log('Exception stack trace:\n\n{}'.format(traceback.format_exc()))
     
     if _shotgun_is_initialized:
-        log('Shotgun has been initialized in the client process')
+        log('Shotgun has been initialized in the bootstrap process')
```
Comment (lassond) [inline plugins/basic/sg_bootstrap.py:91]: Change to `Shotgun has been initialized`

File: plugins/basic/sg_bootstrap.py [hunk 6]
```diff
@@ -88,9 +88,9 @@ def bootstrap_shotgun():
...
     else:
-        log('Shotgun has not been initialized in the client process')
+        log('Shotgun has not been initialized in the bootstrap process')
```
Comment (lassond) [inline plugins/basic/sg_bootstrap.py:93]: Change to `Shotgun has not been initialized`

**Review by lassond — CHANGES_REQUESTED** [review]
Left change requests in comments

**Review by BadSingleton — APPROVED** [review]
LGTM

File: plugins/basic/tk_create_menus/generateMenuItems.py [hunk 7]
```diff
@@ -27,17 +27,29 @@ def __init__(
...
 {{
     class {className}
     {{
+        public static void Call(string menuItem)
+        {{
+            PythonRunner.EnsureInitialized();
+            using (Py.GIL())
+            {{
+                dynamic sg_bootstrap = PythonEngine.ImportModule("sg_bootstrap");
+                sg_bootstrap.test_execute_menu_item(menuItem);
```
Comment (lassond) [inline plugins/basic/tk_create_menus/generateMenuItems.py:49]: here is another "test_" method name.. Could you rename it? No need for me to review again if you change it

Comment (lassond) [reply plugins/basic/tk_create_menus/generateMenuItems.py:49]: I love that you refactored that part by the way :-)

**Pull request merged**